src/Catty/Store/StoreProjectUploader.swift

```diff
@@ -66,52 +66,35 @@ final class StoreProjectUploader: StoreProjectUploaderProtocol {
     }
 
     func upload(project: Project, completion: @escaping (String?, StoreProjectUploaderError?) -> Void, progression: ((Float) -> Void)?) {
+
         guard let zipFileData = fileManager.zip(project) else { completion(nil, .zippingError); return }
 
         let checksum = zipFileData.md5()
+        let attachments = [AttachmentData(name: uploadParameterTag, data: zipFileData, filename: ".zip")]
 
         let uploadUrl = NetworkDefines.uploadUrl
         let urlString = "\(uploadUrl)/\(NetworkDefines.connectionUpload)"
+        let url = URL(string: urlString)!
```

Comment:
Please do not use force-unwrapping in production code. Use the completion handler to let the caller know that an error occurred (i.e., `.unexpectedError`).